data_structures/src/Stack/Stack.cs

```diff
@@ -6,44 +6,71 @@ namespace ADT
 {
     public class Stack<T> : IStack<T>
     {
+        private T[] _stack { get; set; }
```

Comment:
Call this `items` to make it clear what it is. Your internal representation of the data is not equal to the behavior of your data structure - in this case, a stack.